rcl_logging_spdlog/src/rcl_logging_spdlog.cpp

```diff
@@ -13,14 +13,15 @@
 // limitations under the License.
 
 #include <cerrno>
+#include <cassert>
```

Comment:
Why the addition of this header?

Reply:
To get an error code with a parameter without throwing an exception at L142-144.
I will change it to <system_error> for a more narrow scope.